### ISSUE TITLE
[BUG] FVTT KeyboardManager doesn't get modifiers if button glow activated before modifier key

### ISSUE THREAD

**Issue by farling42** [Larkinabout/fvtt-token-action-hud-core#50]
When I move my mouse into a HUD button it gets a glow.

If I hold down ALT before moving the mouse into the button to create the glow, then a mouse click honours the ALT key.

If I move the mouse into the button first to create the glow, then press ALT and then do the mouse click, then the ALT key is ignored.

It works properly in the APP, but not in the Chrome browser.

If I click on a category name, then click on a subcategory name, the ALT key is completely ignored. i.e. the ALT key state is not captured by FVTT KeyboardManager.

---

The game system (Cypher system) calls `game.keyboard.isModifierActive('Alt')` to see if the Alt key is currently held down.

It seems that the glow effect of the HUD buttons is preventing the core FVTT from detecting the ALT key being pressed.

**Comment by Larkinabout** [comment]
The glow is just css, so it's unlikely to be the culprit. It may be a positional cursor issue, though nothing springing to mind as to why that might be. I'll investigate further.

**Comment by farling42** [comment]
Could this be related to you using Pixi to emulate buttons, and so the KeyboardManager isn't properly receiving the ALT key events?
Kim posted in the discord v10-feedback channel:
```
Probably not a case of foundry not being the first to process the events, more likely that releaseKeys is being fired on mouse interaction with a form element as part of this fix https://github.com/foundryvtt/foundryvtt/issues/6978.
```

**Comment by Larkinabout** [comment]
Yeah, I'd just resolved to the same. Foundry VTT triggers a 'focusin' event that releases keys for HTML elements, including buttons, so as soon as you click on the button, it focuses in on it, the event releases the keys, then the action is handled.

The ultimate option might be to change it from a button, though I'd like to explore options of bypassing or perhaps reapplying the key after the focusin event has removed it. The keys are all logged in the event, so I could probably re-add the values to the KeyboardManager.

**Comment by farling42** [comment]
> The ultimate option might be to change it from a button, though I'd like to explore options of bypassing or perhaps reapplying the key after the focusin event has removed it. The keys are all logged in the event, so I could probably re-add the values to the KeyboardManager.

That sounds like a good idea.

**Comment by farling42** [comment]
Many thanks on the rapid fix :)

**Comment by Larkinabout** [comment]
Fixed in Token Action HUD Core 1.2.1.
